lib/tasks/pictograms.rake

```diff
@@ -127,7 +127,7 @@ namespace :pictograms do
       file_path = File.join tmp_load_folder, pictogram[:name]
       File.open(file_path, 'wb') do |file|
         puts "Downloading: #{pictogram[:url]}"
-        file.write open(pictogram[:url]).read
+        file.write open(pictogram[:url], :proxy => ENV['http_proxy'].blank? ? nil : ENV['http_proxy']).read
```

Comment:
Good catch!
Nit: Please, use new Hash syntax: ```{ key: value }``` instead of ```{ :key => value }```.

Reply:
Great! Done!